config/routes.rb

```diff
@@ -3,6 +3,10 @@
 
   mount ActionCable.server => '/cable'
 
+  namespace :maintaining do
+    get 'iterations', to: 'iterations_summary_table#index', as: 'iterations_summary_table'
```

Comment:
So, this should be:
```
resources :iterations, only: [:index]
```

That will then create a route called `maintaining_iterations_path`, which is `/maintaining/iterations`, which maps to the `index` method (and template) in the `maintaining/iterations` controller.

Reply:
You can then change it to `[:index, :show]` to add a `show method` which maps to `/maintaining/iterations/:id`, which will be called `maintaining_iteration_path` (singular) which is a method that expects an id of the object to route to.

Reply:
The code you've got here is good for arbitary one-off routes that don't follow the normal RESTful approach Rails takes (for example, the test routes)

Reply:
Nice!